daemon/openrazer_daemon/dbus_services/dbus_methods/chroma_keyboard.py

```diff
@@ -402,10 +433,14 @@ def set_breath_triple_effect(self, red1, green1, blue1, red2, green2, blue2, red
     :param blue3: Blue component
     :type blue3: int
     """
-    self.logger.debug("DBus call set_breath_dual_effect")
+    self.logger.debug("DBus call set_breath_triple_effect")
 
     # Notify others
-    self.send_effect_event('setBreathDual', red1, green1, blue1, red2, green2, blue2, red3, green3, blue3)
+    self.send_effect_event('setBreathTriple', red1, green1, blue1, red2, green2, blue2, red3, green3, blue3)
+
+    # remember effect
+    self.current_effect = 'breathTriple'
+    self.current_effect_colors = int(red1), int(green1), int(blue1), int(red2), int(green2), int(blue2), int(red3), int(green3), int(blue3)
```

Comment:
Shoudln't it be `self.current_effect_colors[0:9] = ` ?

Reply:
I thought the = operator assigned? The size of the current_effect_colors list is implicit, 9.